docs/assets/css/site.css

```diff
@@ -0,0 +1,520 @@
+    padding: 4px 20px;
+    color: #b8ffd8;
+}
+
+.top-header .container .menu .menu-item.active {
+    background-color: #b8ffd8;
+}
+
+.top-header .container .menu .menu-item.active a {
+    color: #ababab;
+}
+
+.top-header .container .menu .menu-item.active:hover a {
+    color: #3d6850;
+}
+
+.top-header .container .menu .menu-item a {
+    font-size: 16px;
+    opacity: 1;
+    color: #b8ffd8;
+    transition: color 0.3s ease-in-out;
+    margin: 0;
+    height: 44px;
+    line-height: 44px;
+}
+
+.top-header .container .menu .menu-item a:hover {
+    color: #fff;
+}
+
+.top-header .menu-toggle {
+    top: 10px;
+}
+
+@media (max-width: 992px) {
+    .top-header .container .menu {
+        top: 52px;
+        background-color: #171717;
+    }
+}
+
+/*
+    Splash Page
+*/
+
+.splash-page {
+    background-color: #e5e5e5;
+}
+
+.splash-page .splash-header {
+    padding-top: 54px;
+    background: #000;
+    box-shadow: none;
+    height: 500px;
+    border-bottom: 1px solid #0a0f15;
+}
+
+.splash-page .splash-header .caf-brand {
+    background: url(../img/autoscaler-header.png), url(../img/autoscaler-header-image-2.png), #000;
+    background-repeat: no-repeat;
+    background-size: contain;
+    background-position: left bottom;
+    padding-right: 15px;
+    padding-left: 15px;
+    margin-right: auto;
+    margin-left: auto;
+    height: 100%;
+
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.splash-page .splash-header .caf-brand .title {
+    font-size: 50px;
+    text-transform: uppercase;
+    margin-bottom: 20px;
+}
+
+.splash-page .splash-header .caf-brand .slogan {
+    font-family: Metric;
+}
+
+@media (min-width: 768px) {
+    .splash-page .splash-header .caf-brand {
+        width: 750px;
+    }
+}
+
+@media (min-width: 992px) {
+    .splash-page .splash-header .caf-brand {
+        width: 970px;     
+    }
+}
+
+@media (min-width: 1200px) {
+    .splash-page .splash-header .caf-brand {
+        width: 1400px;
+        justify-content: flex-end;        
+    }
+}
+
+
+.splash-page .splash-header .caf-brand .download-caf {
+    background-color: #b8ffd8;
+    color: #fff;
+    padding: 15px 65px;
+    height: auto;
+    margin-top: 0;
+    transition: background-color 0.3s ease-in-out;
+}
+
+.splash-page .splash-header .caf-brand .download-caf:focus, .splash-page .splash-header .caf-brand .download-caf:visited, .splash-page .splash-header .caf-brand .download-caf:active {
+	color: #000;
+}
+
+.splash-page .splash-header .caf-brand .download-caf:hover {
+    background-color: #6EAF8C;
+}
+
+.spash-page {
+	background-color: #e5e5e5;
+}
+.splash-page hr {
+    border-top: 1px solid #323d43;
+    max-width: 1170px;
+    margin: 0 auto;
+    width: 100%;
+}
+
+.splash-page .social-card .icon {
+    color: #74E291;
+}
+
+.splash-page .social-card .title {
+    color: #000;
+}
+
+.splash-page .social-card .subtitle {
+    color: #000;
+}
+
+.splash-page .social-card .link {
```

Comment:
I understand it's colour "#b8ffd8" to be consistent with the other site's, but I would change this to a darker colour so it stands out better against the grey background